doc/sphinx/source/community/review.rst

```diff
@@ -0,0 +1,170 @@
+.. _reviewing:
+
+Contributing a pull request review
+==================================
+
+In the ESMValTool community we use pull request reviews to ensure all code and
+documentation contributions are of good quality.
+An introduction to code reviews can be found in `The Turing Way`_, including
+`why code reviews are important`_ and advice on
+`how to have constructive reviews`_.
+
+Most pull requests will need two reviews before they can be merged.
+First a technical review takes place and then a scientific review.
+Once both reviewers have approved a pull request, it can be merged.
+These three steps are described in more detail below.
+If a pull request contains only technical changes, e.g. a pull request that
+corrects some spelling errors in the documentation or a pull request that
+fixes some installation problem, a scientific review is not needed.
+
+If you are a regular contributor, please try to review a bit more than two
+other pull requests for every pull request you create yourself, to make sure
+that each pull request gets the attention it deserves.
+
+
+1. Technical review
+-------------------
+
+Technical reviews are done by the technical review team.
+This team consists of regular contributors that have a strong interest and
+experience in software engineering.
+
+Technical reviewers use the technical checklist from the
+`pull request template`_ to make sure the pull request follows the standards we
+would like to uphold as a community.
+The technical reviewer also keeps an eye on the design and checks that no major
+design changes are made without the approval from the technical lead development
+team.
+If needed, the technical reviewer can help with programming questions, design
+questions, and other technical issues.
+
+The technical review team can be contacted by writing
+`@ESMValGroup/tech-reviewers`_ in a comment on an issue or pull request on
+GitHub.
+
+2. Scientific review
+--------------------
+
+Scientific reviews are done by the scientific review team.
+This team consists of contributors that have a strong interest and
+experience in climate science.
```

Comment:
This may a bit strict, we also have a few hydrology recipes.

Reply:
Do you have a suggestion for a better wording?

Reply:
Something like Climate-related domains?

Reply:
"climate science or related domains"?